examples/ecs_with_ec2/main.tf

```diff
@@ -0,0 +1,100 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# This is example 2 showcasing Terra3's capabilities.
+#
+# Outcome: Like example 1 + a container runtime and no custom domain
+# ---------------------------------------------------------------------------------------------------------------------
+
+locals {
+  solution_name = "terra3-ecs"
+}
+
+module "terra3_examples" {
+  source = "../.."
+
+  solution_name                 = local.solution_name
+  enable_account_best_practices = true
+
+  # configure your environment here
+  create_load_balancer = true
+
+  # dependency: required for downloading container images
+  nat = "NAT_INSTANCES"
+
+  #cluster_type = "ECS_FARGATE"
+  #cluster_type = "ECS_EC2"
+
```

Comment:
Please make the EC2 cluster configurable from here:
- cluster_ec2_min_nodes (default: 1)
- cluster_ec2_max_nodes (default: 2)
- cluster_ec2_instance_type (default: t3a.small)
- cluster_ec2_desired_capacity (default: 1)
- cluster_ec2_detailled_monitoring (default: false)

Reply:
I did not understand, what I should do here?

Reply:
A developer might want to change the instance type of an ECS EC2-based cluster. Or modify the desired capacity. This is currently not possible. To enable it the Terra3 module needs to have these variables in place and forward these to the cluster module. Does this bring some clarity to your question?

Reply:
1) While using cluster_ec2_volume_size (default: 20gb), it shows the error saying "ValidationError: You must use a valid fully-formed launch template. Volume of size 20GB is smaller than snapshot 'snap-00cb66385216d7ce6', expect size >= 30GB". 
Due to that reason I have used 30 GB as default.